src/sentry/static/sentry/app/components/organizations/timeRangeSelector/utils.tsx

```diff
@@ -10,11 +10,11 @@ const DATE_TIME_FORMAT = 'YYYY-MM-DDTHH:mm:ss';
  * and end date, with the end date as the current time and the start date as the
  * time that is the current time less the statsPeriod.
  *
- * @param {String} statsPeriod Relative stats period
- * @return {Object} Object containing start and end date as YYYY-MM-DDTHH:mm:ss
+ * @param statsPeriod Relative stats period
+ * @return Object containing start and end date as YYYY-MM-DDTHH:mm:ss
```

Comment:
Please is the T in between correct?

Reply:
I guess so, format is above.
```ts
const DATE_TIME_FORMAT = 'YYYY-MM-DDTHH:mm:ss';
```

Reply:
Oh I didn't see it before...it should be fine then :)